dev/service-registry-client/src/main/scala/com/lightbend/lagom/internal/registry/ServiceRegistryModule.scala

```diff
@@ -34,13 +34,6 @@ class ServiceRegistryModule(environment: Environment, configuration: Configurati
           "bound to an implementation that will query the embedded Service Locator. This is fine to use " +
           "only during development."
       }
-    } else if (environment.mode == Mode.Test) {
```

Comment:
Because if you're running in test, and you actually do want to bind a service locator for the tests to use, binding one here gives you a duplicate binding.